airflow/ui/src/components/AppContainer/TimezoneDropdown.tsx

```diff
@@ -44,9 +44,9 @@ const TimezoneDropdown: React.FC = () => {
   const timezones = getTimeZones();
 
   let currentTimezone;
-  const options = timezones.map(({ name, currentTimeFormat }) => {
+  const options = timezones.map(({ name, currentTimeFormat, group }) => {
     const label = `${currentTimeFormat.substring(0, 6)} ${name.replace(/_/g, ' ')}`;
-    if (name === timezone) currentTimezone = { label, value: name };
+    if (name === timezone || group.includes(timezone)) currentTimezone = { label, value: name };
```

Comment:
Does this let you search for "America/San Diego" and get LA for instance?

Reply:
I'm not quite sure what this code is for then? If the only option is what is in the drop down, then we can ignore groups can't we?

Reply:
<img width="354" alt="Screen Shot 2021-05-07 at 10 34 21" src="https://user-images.githubusercontent.com/4600967/117465519-68a9d000-af17-11eb-958d-19c0696e9ea0.png">

If the guessed timezone is "America/San Diego" then it will resolve it to show LA instead of being null.

Reply:
Got it!